client/.gitignore

```diff
@@ -0,0 +1,47 @@
+# Miscellaneous
+*.class
+*.log
+*.pyc
+*.swp
+.DS_Store
+.atom/
+.buildlog/
+.history
+.svn/
+migrate_working_dir/
+
```

Comment:
```suggestion
# Django stuff:
*.log
local_settings.py
db.sqlite3
```

Reply:
the `server/db.sqlite3` should not be in git.
This will avoid future addition but now that it's in you will have to `git rm` it

Reply:
Same here too?